sqrl-calcite/src/main/java/com/datasqrl/plan/validate/ScriptPlanner.java

```diff
@@ -155,35 +155,29 @@ public Void visit(SqrlCreateDefinition statement, Void context) {
     NamePath namePath = NamePath.of(statement.getName().names.toArray(String[]::new));
     String logId = statement.getName().names.get(0);
 
-    Map map = new HashMap();
-    map.put("primary-key", List.of("_uuid"));
-    map.put("timestamp-type", "LOG_TIME");
-    map.put("timestamp-name", "event_time");
-    map.put("name", logId);
-
-    Optional<TableConfig> sink = connectorFactoryFactory.create(Type.LOG, "kafka")
-        .map(t->t.createSourceAndSink(new ConnectorFactoryContext(logId, map)));
-
-    if (sink.isEmpty()) {
-      addError(ErrorCode.CANNOT_RESOLVE_TABLESINK, statement, "Cannot resolve log: %s",
-          logId);
-      return null;
-    }
-
-    TableConfig tableConfig = sink.get();
-    this.framework.getSchema().add(new ResolvedImport(logId, tableConfig));
     TypeFactory typeFactory = TypeFactory.getTypeFactory();
     RelDataTypeBuilder fieldBuilder = CalciteUtil.getRelTypeBuilder(typeFactory);
     fieldBuilder.add(Name.system("_uuid"), typeFactory.createSqlType(SqlTypeName.VARCHAR));
-//    fieldBuilder.add(Name.system("event_time"), typeFactory.createSqlType(SqlTypeName.TIMESTAMP_WITH_LOCAL_TIME_ZONE, 3));
+    // TODO: that's an ugly hack, postgres-log requires it as a field, however kafka requires it as metadata
+    if (logManager.getEngineName().equals("postgres_log")) {
```

Comment:
Must be removed before landing. There are too many if conditions here.

Reply:
On it.